packages/sdk-client/src/api/api-client.ts

```diff
@@ -77,7 +77,7 @@ export class ApiClient {
           const prop = data[name];
           acc[name] = typeof prop.toJSON === 'function'
             ? prop.toJSON()
-            : Array.isArray(prop) ? prop.join(';') : prop.toString();
+            : Array.isArray(prop) ? prop.join(',') : prop.toString();
```

Comment:
Does this fit if any array item contains a comma character? 
This could have side effect to decode it later. Right ?